code/controllers/subsystem/mapping.dm

```diff
@@ -0,0 +1,37 @@
+SUBSYSTEM_DEF(mapping)
+   	name = "Mapping"
+   	init_order = 12
+   	flags = SS_NO_FIRE
+
+/datum/controller/subsystem/mapping/New()
+   	NEW_SS_GLOBAL(SSmapping)
+   	return ..()
+
+/datum/controller/subsystem/mapping/Initialize(timeofday)
+   	// Load all Z level templates
```

Comment:
Dont know why this is indented weird, its not like that in my file
![image](https://user-images.githubusercontent.com/25063394/54865371-63e82900-4d5c-11e9-84e5-f4a6b9c5a70b.png)

Reply:
this file is literally all spaced indentation.